annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -65,11 +67,24 @@ public class AnnotationServiceImpl extends BaseAnnotationServiceImpl implements
   @Autowired
   @Qualifier(AnnotationConfiguration.BEAN_METIS_DEREFERENCE_CLIENT)
   MetisDereferenciationClient dereferenciationClient;
-
+  
+  private HttpConnection httpConnection;
```

Comment:
better create a new class SearchApiClient, to handle the information aquisition through the search api